frontend/src/routes/manage/index.tsx

```diff
@@ -133,6 +133,7 @@ export const ManageNav: React.FC<ManageNavProps> = ({ active }) => {
     const entries: [NonNullable<ManageNavProps["active"]>, string, ReactElement][] = [
         ["/~manage", t("manage.nav.dashboard"), <HiOutlineTemplate />],
         ["/~manage/videos", t("manage.nav.my-videos"), <FiFilm />],
+        ["/~upload", t("upload.title"), <FiUpload />],
```

Comment:
Of course now the URL path doesn't quite correspond to the breadcrumbs anymore... mh. Should we change that? Does it matter?

Reply:
Hm I don't think it does. I mean we could remove the `mangement` part of the breadcrumbs, but that would in turn somewhat mismatch the other item (`My videos`) that is reachable from the management dashboard (which on the other hand wouldn't be a big deal either, so I'm struggling to make a decision here).

Reply:
I was rather thinking to change the URL of the uploader to `/~manage/upload`. And after thinking more and asking Julian, I would slightly prefer if yes we would change that. Can you do that still? (Also remember to catch all places where it is used)